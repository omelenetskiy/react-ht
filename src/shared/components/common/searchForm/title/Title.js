import React from 'react';

import StyledTitle from '../../../styled/title';

const Title = ({ children }) => <StyledTitle>{children}</StyledTitle>;

export default Title;
