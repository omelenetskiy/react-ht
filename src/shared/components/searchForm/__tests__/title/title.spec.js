import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-enzyme';
import 'jest-styled-components';

import Title from '../../title/Title';

const props = {
  children: 'title',
};

describe('Title', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Title {...props} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
