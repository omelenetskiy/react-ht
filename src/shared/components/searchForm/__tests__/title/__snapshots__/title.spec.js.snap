// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render correctly 1`] = `
<title___default>
  title
</title___default>
`;
